add DALES simulations botany_dx100m_nx96

fjansson opened the pull request:
similar to the previous botany_dx_100m_nx1536, except:
* the domain size is smaller, 96 cells horizontally
* the simulation time is longer, 7 days.

Test script, to access the new datasets (loading catalog from my fork of the eurec4a-intake catalog):
```python
import matplotlib.pyplot as plt
import pandas as pd
from intake import open_catalog

url ="https://raw.githubusercontent.com/fjansson/eurec4a-intake/botany-96/Simulations/DALES/botany.yaml"
cat = open_catalog(url)

parameters = cat.dx100m.nx96.parameters.read()
varied_parameters = ['member','thls', 'u0', 'qt0', 'qt_lambda', 'thl_Gamma', 'wpamp', 'dudz', 'location']
df_parameters = pd.DataFrame.from_records(parameters)[varied_parameters]
print(df_parameters)

ds_2D = cat.dx100m.nx96['2D'].to_dask()
print(ds_2D)

ds_3D = cat.dx100m.nx96['3D'].to_dask()
print(ds_3D)

ds_radiation = cat.dx100m.nx96['radiation'].to_dask()
print(ds_radiation)

ds_cross_xy = cat.dx100m.nx96['cross_xy'].to_dask()
print(ds_cross_xy)

ds_profiles = cat.dx100m.nx96['profiles'].to_dask()
print(ds_profiles)

ds_timeseries = cat.dx100m.nx96['timeseries'].to_dask()
print(ds_timeseries)

# plot a 2D field
ds_2D.sel(member=10).isel(time=400).twp.plot()

# plot same field from the larger botany simulation
plt.figure()
ds_2D_large = cat.dx100m.nx1536['2D'].to_dask()
ds_2D_large.sel(member=10).isel(time=400).twp.plot()
plt.show()
```

observingClouds reviewed (commented):
Hi @fjansson,
Great to see your additional simulations pop-up here. I will still need to check the entires, but since the CI is succeeding they are likely fine. Could you address in the meantime my comment regarding the dataset description?
Thank you!

observingClouds reviewed (commented):
We are not super consistent in the catalog about starting a description with a capital letter, but most descriptions are, so let's improve on that.

fjansson commented:
Thanks for the comments!

I amended the commit:
* In botany_dx100m.yaml I differentiate the two datasets by mentioning the domain size.
* I fixed the capitalization, also the old botany-1536.
* I changed Timeseries and Profiles to "Timeseries of domain means",  "Profiles, horizontally averaged" to be more descriptive (since our profiles aren't single columns but horizontal averages of the domain).

observingClouds reviewed (approved):
Looks all good to me now! Great work! Now it would be great if you could add a few words on the howto.eurec4a book so that people are more aware about it. But I guess you already had that in min.

fjansson commented:
Thanks! Access to the small dataset now works through the eurec4a module, like the larger one. We'll add something to the book too.
